src/redux/book/Bookslice.js

```diff
@@ -0,0 +1,44 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [
+    // {
+    //   item_id: 'item1',
+    //   title: 'The Great Gatsby',
+    //   author: 'John Smith',
+    //   category: 'Fiction',
+    // },
+    // {
+    //   item_id: 'item2',
+    //   title: 'Anna Karenina',
+    //   author: 'Leo Tolstoy',
+    //   category: 'Fiction',
+    // },
+    // {
+    //   item_id: 'item3',
+    //   title: 'The Selfish Gene',
+    //   author: 'Richard Dawkins',
+    //   category: 'Nonfiction',
+    // },
+  ],
+};
```

Comment:
- [ ] to keep your repo clean kindly remove any commented code from your project , either remove it or you can leave it as initialState, kindly fix it 🙏